typescript/web/src/utils/tests/data.fixtures.ts

```diff
@@ -21,6 +26,55 @@ export type LabelClassData = Pick<
   labelsAggregates: LabelsAggregates;
 };
 
+export type LabelData = SetRequired<
+  Pick<
+    Label,
+    | "id"
+    | "type"
+    | "imageId"
+    | "x"
+    | "y"
+    | "width"
+    | "height"
+    | "smartToolInput"
+    | "geometry"
+  >,
+  "smartToolInput"
+> & {
+  labelClass: LabelClassData;
+};
+
+export type ImageData = SetRequired<
+  Pick<Image, "id" | "name" | "url" | "thumbnail200Url" | "width" | "height">,
+  "thumbnail200Url"
+> & {
+  dataset: DatasetData;
+  labels: (Omit<LabelData, "imageId" | "labelClass"> & {
+    labelClass: Omit<LabelClassData, "dataset">;
+  })[];
+};
+
+const BASIC_LABEL_GEOMETRY_PARTIAL = {
```

Comment:
Why is there a `BASIC_` prefix?  
What makes the data being Partial?  
Why is there no TypeScript type?